charts/tigera-operator/values.yaml

```diff
@@ -0,0 +1,30 @@
+imagePullSecrets: {}
+
+installation:
+  enabled: true
+  kubernetesProvider: ""
+
+apiServer:
+  enabled: true
+
+certs:
+  node:
+    key:
+    cert:
+    commonName:
+  typha:
+    key:
+    cert:
+    commonName:
+    caBundle:
+
+resources: {}
+
+# Configuration for the tigera operator
+tigeraOperator:
+  image: tigera/operator
+  version: master
```

Comment:
This used to be auto-generated from a ruby script which would populate the version from _data/versions.yaml

It's a bit unfortunately, but in this PR there's a new step as part of a release to update this file with the proper version. I suppose hashreleases will need to be updated accordingly as well.

The alternative is to add in some other kind of auto generation, but was trying to keep this simple and avoid any cross dependency on the docs